cluster-provision/k8s/provision.sh

```diff
@@ -12,7 +16,17 @@ cd $DIR
 
 export KUBEVIRT_CGROUPV2="${CGROUPV2}"
 
-(cd ../${base} && ./build.sh)
+if [[ $PHASES =~ linux.* ]]; then
+  (cd ../${base} && ./build.sh)
+fi
+
 make -C ../gocli cli
-../gocli/build/cli provision ${provision_dir}
-./check-cluster-up.sh ${provision_dir}
+../gocli/build/cli provision ${provision_dir} --phases ${PHASES}
+
+if [[ $PHASES == $PHASES_DEFAULT ]] || [[ $CHECK_CLUSTER == true ]]; then
+   if [[ $PHASES == "linux" ]]; then
+     echo "skipping cluster check when running linux only phase"
+     exit 0
+   fi
+  ./check-cluster-up.sh ${provision_dir}
+fi
```

Comment:
We can simplify this 

```bash
if  [[ $PHASES =~ "k8s" ]] && [[ "$CHECK_CLUSTER" == true ]]; then
     ./check-cluster-up.sh ${provision_dir}
fi
```

Reply:
nope because if PHASES=k8s you don't always want to check the cluster, just if asked to